File: CS/LoadingPanelExample/Form1.cs
using System;
using DevExpress.XtraEditors;
using System.Windows.Forms;

namespace LoadingPanelExample {
    public partial class Form1 :XtraForm {
        public Form1() {
            InitializeComponent();
        }

        int pos = 0;
        LoadingPanel fLoadingPanel;
        private void OnTimerTick(object sender, EventArgs e) {
            switch (pos) {
                case 0: ShowLoadingPanel(listBox); break;
                case 1: ShowLoadingPanel(radioGroup); break;
                case 2: ShowLoadingPanel(picture); break;
                case 3: ShowLoadingPanel(memo); break;
            }
        }

        void ShowLoadingPanel(Control control) {
            if (fLoadingPanel != null)
                fLoadingPanel.Dispose();
            fLoadingPanel = new LoadingPanel(control, null);
            pos = pos == 3 ? 0 : pos + 1;
        }

        private void OnLoad(object sender, EventArgs e) {
            timer.Start();
        }
    }
}

File: VB/LoadingPanelExample/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraEditors
Imports System.Windows.Forms

Namespace LoadingPanelExample
	Partial Public Class Form1
		Inherits XtraForm
		Public Sub New()
			InitializeComponent()
		End Sub

		Private pos As Integer = 0
		Private fLoadingPanel As LoadingPanel
		Private Sub OnTimerTick(ByVal sender As Object, ByVal e As EventArgs) Handles timer.Tick
			Select Case pos
				Case 0
					ShowLoadingPanel(listBox)
				Case 1
					ShowLoadingPanel(radioGroup)
				Case 2
					ShowLoadingPanel(picture)
				Case 3
					ShowLoadingPanel(memo)
			End Select
		End Sub

		Private Sub ShowLoadingPanel(ByVal control As Control)
			If fLoadingPanel IsNot Nothing Then
				fLoadingPanel.Dispose()
			End If
			fLoadingPanel = New LoadingPanel(control, Nothing)
			If pos = 3 Then
				pos = 0
			Else
				pos = pos + 1
			End If
		End Sub

		Private Sub OnLoad(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			timer.Start()
		End Sub
	End Class
End Namespace